### PR TITLE
ItemData component

### PR DESCRIPTION
**Pull request by Bidjii**

### CODE REVIEW

File: src/item/style.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    padding-top: ${space.l};
+    padding-bottom: ${space.l};
+    align-items: center;
+    flex: 1;
+    border: 0;
+    background: none;
+  }
+
+  /* Text areas */
+  .kirk-item-leftTxt {
```
Comment (Tuxosaurus) [inline src/item/style.ts]: this.props.

File: src/itemData/style.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import css from 'styled-jsx/css'
+
+import { color } from '_utils/branding'
+
+export default css`
+  .kirk-item-data {
```
Comment (Tuxosaurus) [inline src/itemData/style.ts]: I'd remove the file if it's empty

**Comment by loick** [comment]
Tu peux ajouter des screenshots des deux nouveaux composants stp?

File: src/itemData/style.ts (hunk 2 above)
Comment (Zarthia) [reply src/itemData/style.ts]: You're right, didn't notice this one

File: src/text/style.ts [hunk 3]
```diff
@@ -26,7 +26,7 @@ export default css`
     line-height: ${font.s.lineHeight};
   }
 
-  .kirk-text-title,
+  .git kirk-text-title,
```
Comment (Tuxosaurus) [inline src/text/style.ts]: `.git` nani?!

Comment (Bidjii) [reply src/text/style.ts]: dafuk

File: src/_utils/item/index.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+  rightTitle,
+  rightTitleDisplay = TextDisplayType.TITLE,
+  rightBody,
+  rightBodyDisplay = TextDisplayType.BODY,
+  rightAddon,
+  tag = 'div',
+}:ItemProps) => {
+  const Tag = tag
+  const baseClassName = 'kirk-item'
+  const isRightTxt = rightTitle || rightBody
```
Comment (pith) [inline src/_utils/item/index.tsx]: nit: `isRightTxt` => `hasRightText`

File: src/_utils/item/index.unit.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  const wrapper = shallow(<Item leftAddon={<ClockIcon />} />)
+  expect(wrapper.find(ClockIcon).exists()).toBe(true)
+})
+
+it('Should display a right add-on', () => {
+  const wrapper = shallow(<Item rightAddon={<ClockIcon />} />)
+  expect(wrapper.find(ClockIcon).exists()).toBe(true)
+})
+
+it('Should display an arrow', () => {
```
Comment (pith) [inline src/_utils/item/index.unit.tsx]: Arrow or chevron ?

File: src/_utils/item/index.unit.tsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  const wrapper = shallow(<Item tag="li" />)
+  expect(wrapper.type()).toBe('li')
+})
+
+it('Should display a left add-on', () => {
+  const wrapper = shallow(<Item leftAddon={<ClockIcon />} />)
+  expect(wrapper.find(ClockIcon).exists()).toBe(true)
+})
+
+it('Should display a right add-on', () => {
```
Comment (pith) [inline src/_utils/item/index.unit.tsx]: also add a test of `kirk-item-rightText`

File: src/_utils/item/style.ts [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+  /* Text */
+  :global(.kirk-item-title) {
+    margin-bottom: ${space.s};
+  }
+
+  /* Addons */
+  :global(.kirk-item-leftAddon),
+  :global(.kirk-item-rightAddon) {
+    display: inline-flex;
+    min-width: 24px;
```
Comment (pith) [inline src/_utils/item/style.ts]: nit: `24px` => `${space.xl}`

File: src/text/index.tsx [hunk 8]
```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import React, { Fragment } from 'react'
```
Comment (pith) [inline src/text/index.tsx]: nit: `Fragment` is not used

File: src/_utils/item/style.ts (hunk 7 above)
Comment (Bidjii) [reply src/_utils/item/style.ts]: Not sure if we want to change this value if we change the `xl` value at some point

**Comment by Bidjii** [comment]
@pith https://jira.corp.blablacar.com/browse/BBCSPA-1752

**Pull request merged**